docs/how-to/create-mtls-client-credentials.md

```diff
@@ -174,22 +184,44 @@ sudo charmed-kafka.acls --bootstrap-server $KAFKA_SERVERS_SASL --command-config
 
 ## Test access
 
+To test the client's access, first create a file called `client-mtls.properties` with the following configuration:
```

Comment:
Suggestion: Given the `cat` command, it seems like the following is the command to create the file, rather than the contents of the file. Correct me if I'm wrong.

```suggestion
To test the client's access, first create a file called `client-mtls.properties`:
```